tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -23,22 +23,21 @@ Login To RHODS Dashboard
    ${authorize_service_account} =  Is rhods-dashboard Service Account Authorization Required
    Run Keyword If  ${authorize_service_account}  Authorize rhods-dashboard service account
 
-Wait for RHODS Dashboard to Load
-  [Arguments]  ${dashboard_title}="Red Hat OpenShift Data Science Dashboard"
-  Wait For Condition  return document.title == ${dashboard_title}  timeout=15
+Wait for ${ods_project} Dashboard to Load
+  Wait For Condition  return document.title == "${${ods_project}_DASHBOARD_TITLE}"  timeout=15
```

Comment:
See comment about using a common variable name for variables that have different values between ODH and RHODS